### DISCUSSION TITLE
Does this work with Fireblocks?

### DISCUSSION THREAD

**Discussion by seansu4you87** [rainbow-me/rainbowkit#1205] [Q&A]
Hi there, has anyone gotten this working for the Fireblocks wallet? I have an app that works with MetaMask, Rainbow, etc. But when using Fireblocks, I run into issues.

Unfortunately I'm unsure what the root cause is right now, as I'm getting infinite re-render React issues, but I know that the issue is reproducible when I switch to using a Fireblocks wallet over wallet connect. I just wanted to see if anyone in the community has had success with Fireblocks, and if they had to do any custom integration work.

Thanks!

**Comment by thisisdaniel333** [comment]
Have you fixed it? We are running into the exact same issue, and we can identify the eaxct problem

**Reply by DanielSinclair** [reply]
Are you referring to using the Fireblocks mobile app?

**Reply by thisisdaniel333** [reply]
Yes I am
